testing/bdd/features/styles/style_attribute_inherited.feature

```diff
@@ -0,0 +1,44 @@
+@styles @document @inherited
+Feature: Compute style attribute on a single EBU-TT Live element
+
+  Examples:
+  | xml_file                      | cell_resolution | extent      |
+  | style_attribute_inherited.xml | 32 15           | 320px 150px |
+
+
+  # Inheritance: region (S1) > div (S2) > p (S3) > span (S4)
+  Scenario: Inheritable style attributes
+    Given an xml file <xml_file>
+    When it has a cell resolution of <cell_resolution>
+    And it has extent of <extent>
+    And it contains style S1 with <style_attribute> value <S1_value>
+    And it contains style S2 with <style_attribute> value <S2_value>
+    And it contains style S3 with <style_attribute> value <S3_value>
+    And it contains style S4 with <style_attribute> value <S4_value>
+    And the document is generated
+    Then the computed <style_attribute> in <elem_id> is <computed_value>
+
+    Examples:
+    | S1_value  | S2_value | S3_value       | S4_value | style_attribute      | elem_id | computed_value |
+    |           |          |                |          | tts:direction        | span1   | ltr            |
+    | rtl       |          |                |          | tts:direction        | span1   | rtl            |
+    | white     | blue     |                |          | tts:color            | span1   | blue           |
+    |           |          |                |          | tts:color            | span1   |                |
+    |           | serif    | monospaceSerif |          | tts:fontFamily       | span1   | monospaceSerif |
+    |           |          |                |          | tts:fontFamily       | span1   | default        |
+    |           | normal   | italic         |          | tts:fontStyle        | span1   | italic         |
+    |           |          |                |          | tts:fontStyle        | span1   | normal         |
+    |           | bold     | normal         | bold     | tts:fontWeight       | span1   | bold           |
+    |           |          |                |          | tts:fontWeight       | span1   | normal         |
+    | 0.5c      | 0c       | 0.5c           |          | ebutts:linePadding   | span1   | 0.5c           |
```

Comment:
linePadding applies to `body`, `div` and `p` only so testing on a span isn't meaningful.